package.json

```diff
@@ -2,10 +2,10 @@
   "name": "decidim-plataformess",
   "private": true,
   "dependencies": {
-    "@decidim/browserslist-config": "^0.25.2",
-    "@decidim/core": "^0.25.2",
-    "@decidim/elections": "^0.25.2",
-    "@decidim/webpacker": "^0.25.2",
+    "@decidim/browserslist-config": "file:packages/browserslist-config",
```

Comment:
Please, avoid the usage of "file:packages/x". Use "^0.26.5" instead and remove the packages directory